### PR TITLE
PXB-2330: Implement split-job pipeline for PXB 8.0 and little fixes for 2.4

### PR DESCRIPTION
**Pull request by Sudokamikaze**
Builds: https://pxb.cd.percona.com/view/PXB%208.0/job/percona-xtrabackup-8.0-multijob/1/

### CODE REVIEW

File: pxb/v2/jenkins/percona-xtrabackup-8.0-compile-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        default: "https://github.com/percona/percona-xtrabackup"
+        description: URL to percona-xtrabackup repository
+    - string:
+        name: BRANCH
+        default: "8.0"
+        description: Tag/Branch for percona-xtrabackup repository
+    - choice:
+        name: DOCKER_OS
+        choices:
+        - centos:7
```
Comment (EvgeniyPatlan) [inline pxb/v2/jenkins/percona-xtrabackup-8.0-compile-pipeline.yml:28]: сcentos6?

File: pxb/v2/jenkins/percona-xtrabackup-8.0-test-param.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        name: CMAKE_BUILD_TYPE
+        type: user-defined
+        values:
+          - RelWithDebInfo
+          - Debug
+    - axis:
+        type: user-defined
+        name: DOCKER_OS
+        values:
+        - centos:7
```
Comment (EvgeniyPatlan) [inline pxb/v2/jenkins/percona-xtrabackup-8.0-test-param.yml:50]: centos6?

File: pxb/v2/jenkins/percona-xtrabackup-8.0-test-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            branches:
+            - "PXB-2330-pxb80"
+            wipe-workspace: false
+      lightweight-checkout: true
+      script-path: pxb/v2/jenkins/percona-xtrabackup-8.0-test-pipeline.groovy
+    parameters:
+    - choice:
+        name: DOCKER_OS
+        choices:
+          - centos:7
```
Comment (EvgeniyPatlan) [inline pxb/v2/jenkins/percona-xtrabackup-8.0-test-pipeline.yml:20]: centos6

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: pxb/v2/jenkins/percona-xtrabackup-8.0-compile-pipeline.yml (hunk 1 above)
Comment (Sudokamikaze) [reply pxb/v2/jenkins/percona-xtrabackup-8.0-compile-pipeline.yml:28]: Will be added and re-tested, thanks! :)

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**